spec/model/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject(:user) do
+    described_class.new(
+      name: 'Anything',
+      photo: 'http://rrrrrrrr/anything.jpg',
+      bio: 'Anything test',
+      postscounter: 0
+    )
+  end
+
+  it 'name must not be blank' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'postcounter must be an integer greater than or equal to zero' do
+    subject.postscounter = -1
```

Comment:
- [ ] Great job so far. Please, if you check your guidelines well, it says you should add unit specs for all of your models' methods and validations. Please kindly add test for the `Three most recent post` method in the user model